gcloud-java-contrib/pom.xml

```diff
@@ -19,11 +19,7 @@
     <module>gcloud-java-nio</module>
   </modules>
   <dependencies>
-    <dependency>
-      <groupId>${project.groupId}</groupId>
-      <artifactId>gcloud-java</artifactId>
-      <version>${project.version}</version>
-    </dependency>
+
```

Comment:
Nit: I'd actually just remove the <dependencies> tags altogether, no real use having them around.